### PR TITLE
2023 Survey results

### PR DESCRIPTION
**Pull request by demirkayaender**
Adding a new blog post about the survey results

### CODE REVIEW

File: src/blog/_posts/2023-06-08-survey-results.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: 2023 Cadence Community Survey Results
+
+date: 2023-06-08
+author: Ender Demirkaya
+authorlink: https://www.linkedin.com/in/enderdemirkaya/
+---
+
+We have released a user survey earlier this year to learn about who our users were, how they use Cadence and how we could help them. After collecting your great feedback, we wanted to share the results with our community. Thank you everyone for filling it out! Your feedback is invaluable to us and it helps us shape our roadmap for the future.
```
Comment (allenchen2244) [inline src/blog/_posts/2023-06-08-survey-results.md:9]: We released a user survey earlier this year to learn about who our users are, how they use Cadence, and how we can help them. After collecting your great feedback, we wanted to share the results with our community. Thank you everyone for filling it out! Your feedback is invaluable and it helps us shape our roadmap for the future.

File: src/blog/_posts/2023-06-08-survey-results.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Here are some highlights in text and you can check out the visuals to get more details:
+
+![using.png](../../shared/img/2023-survey-results/using.png)
+
+![job_role.png](../../shared/img/2023-survey-results/job_role.png)
+
+Most of the people who replied to our survey were engineers who were already using Cadence, actively evaluating or migrating from a similar technology. This was exciting to hear! Some of you have contacted us to learn more about benchmarks, scale and ideal use cases. We will share more guidelines about this but until then, feel free to contact us over our Slack workspace to get any guidance.
```
Comment (allenchen2244) [inline src/blog/_posts/2023-06-08-survey-results.md:17]: Most of the people who replied to our survey were engineers who were already using Cadence, actively evaluating, or migrating from a similar technology. This was exciting to hear! Some of you have contacted us to learn more about benchmarks, scale, and ideal use cases. We will share more guidelines about this but until then, feel free to contact us over our Slack workspace for guidance.

File: src/blog/_posts/2023-06-08-survey-results.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+![scale.png](../../shared/img/2023-survey-results/scale.png)
+
+Scale our users operating Cadence varies from thousands to billions of workflows per month. It was exciting to see it being used in both small and large scale companies.
```
Comment (allenchen2244) [inline src/blog/_posts/2023-06-08-survey-results.md:21]: The scale*

File: src/blog/_posts/2023-06-08-survey-results.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+![time_zone.png](../../shared/img/2023-survey-results/time_zone.png)
+
+Our user base is the biggest in Europe compared to any other place. Several of those users contacted us to contribute to Cadence as a follow up to the survey. We will start putting up-for-grabs tasks on Github. Several of them wanted to meet with a Zoom call and to discuss their use cases and best practices. As the Cadence team has presence in both the EU and the US, we welcome all our users to contact us anytime. Slack is the fastest way to reach.
```
Comment (allenchen2244) [inline src/blog/_posts/2023-06-08-survey-results.md:25]: Slack is the fastest way to reach us.

File: src/blog/_posts/2023-06-08-survey-results.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+![following.png](../../shared/img/2023-survey-results/following.png)
+
+![channels.png](../../shared/img/2023-survey-results/channels.png)
+
+Cadence is followed in [Slack](https://uber-cadence.slack.com/) the most, then [Github](https://github.com/uber/cadence) and [LinkedIn](https://www.linkedin.com/company/cadenceworkflow/). We are the most active in Slack and we plan to be more active in other mediums as well.
+
+![scenarios.png](../../shared/img/2023-survey-results/scenarios.png)
+All of our main use cases were used across the board. While we mentioned the most common cases, several others were mentioned as a comment: enhanced timers, leader election etc.
+
+We found out that Cadence has been used in several science communities. Some of them were using community built clients and were asking if we are going to support more languages. We are planning to take the ownership of Python and Javascript/Typescript clients and support them officially.
```
Comment (allenchen2244) [inline src/blog/_posts/2023-06-08-survey-results.md:36]: We are planning to take ownership of the Python and Javascript/Typescript clients and support them officially

File: src/blog/_posts/2023-06-08-survey-results.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+![improvement.png](../../shared/img/2023-survey-results/improvement.png)
+
+Documentation is by far what our users wanted improvements on. We are revamping our documentation soon, and expect to see major changes in our website soon.
```
Comment (allenchen2244) [inline src/blog/_posts/2023-06-08-survey-results.md:40]: Documentation is by far what our users wanted improvements on. We are revamping our documentation and there will be major changes on our website soon.

File: src/blog/_posts/2023-06-08-survey-results.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+![help_stage.png](../../shared/img/2023-survey-results/help_stage.png)
+
+Other requests were about observability, debuggability, operability and usability. These areas have been our main focus this year and we are planning to release updates and blogs about them.
+
+![support.png](../../shared/img/2023-survey-results/support.png)
+
+We noticed most of our users need help once a month or more. While we welcome questions and discussions over the mediums mentioned above, we plan to make more public posts about the common issues using our blog, StackOverflow, LinkedIn or Twitter.
```
Comment (allenchen2244) [inline src/blog/_posts/2023-06-08-survey-results.md:48]: same here

File: src/blog/_posts/2023-06-08-survey-results.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+![scale.png](../../shared/img/2023-survey-results/scale.png)
+
+The scale our users operating Cadence varies from thousands to billions of workflows per month. It was exciting to see it being used in both small and large scale companies.
+
+![time_zone.png](../../shared/img/2023-survey-results/time_zone.png)
+
+Our user base is the biggest in Europe compared to any other place. This is inline with Cadence team growing its presence in Europe. Users from different places also contacted us to contribute to Cadence as a follow up to the survey. We will start putting up-for-grabs tasks on Github. Several of them wanted to meet with a Zoom call and to discuss their use cases and best practices. As the Cadence team has presence in both the EU and the US, we welcome all our users to contact us anytime. Slack is the fastest way to reach us.
```
Comment (davidporter-id-au) [inline src/blog/_posts/2023-06-08-survey-results.md:26]: Nit: `This is in-line with the Cadence team` 

`user-base` here refers to distinct respondees presumably, not customers, which would probably be the US?

Comment (davidporter-id-au) [reply src/blog/_posts/2023-06-08-survey-results.md:26]: Up-for-grabs tasks, or `new-starter` tasks?

File: src/blog/_posts/2023-06-08-survey-results.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
+---
+title: 2023 Cadence Community Survey Results
+
+date: 2023-06-08
+author: Ender Demirkaya
+authorlink: https://www.linkedin.com/in/enderdemirkaya/
+---
+
+We released a user survey earlier this year to learn about who our users are, how they use Cadence, and how we can help them. After collecting the feedback, we wanted to share the results with our community. Thank you everyone for filling it out! Your feedback is invaluable and it helps us shape our roadmap for the future.
+
```
Comment (davidporter-id-au) [inline src/blog/_posts/2023-06-08-survey-results.md:10]: I would add a line about your survey methodology, I assume this is mostly gathered from slack however. I get that we're not in the business of doing scientific-surveys, but it helps contextualize the discussion and limitations from what we can understand here.

File: src/blog/_posts/2023-06-08-survey-results.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+Here are some highlights in text and you can check out the visuals to get more details:
+
+![using.png](../../shared/img/2023-survey-results/using.png)
+
+![job_role.png](../../shared/img/2023-survey-results/job_role.png)
+
+Most of the people who replied to our survey were engineers who were already using Cadence, actively evaluating, or migrating from a similar technology. This was exciting to hear! Some of you have contacted us to learn more about benchmarks, scale, and ideal use cases. We will share more guidelines about this but until then, feel free to contact us over our Slack workspace for guidance.
```
Comment (davidporter-id-au) [inline src/blog/_posts/2023-06-08-survey-results.md:18]: Is this is a survey skew result? Nothing wrong with it, but I'm guessing we've selected for folks already using it by surveying from folks in the cadence slack and similar channels

**Review by davidporter-id-au — APPROVED** [review]

File: src/blog/_posts/2023-06-08-survey-results.md (hunk 10 above)
Comment (demirkayaender) [reply src/blog/_posts/2023-06-08-survey-results.md:18]: Yeah most people who follow us are engineers but the survey was shared from Slack, Blog and LinkedIn with multiple reshares by other people.

File: src/blog/_posts/2023-06-08-survey-results.md (hunk 9 above)
Comment (demirkayaender) [reply src/blog/_posts/2023-06-08-survey-results.md:10]: Added

**Pull request merged**